README.md

```diff
@@ -1,45 +1,94 @@
-# Nox.Reference
+Nox.Refence is a storage which contains the most common used types.
```

Comment:
why remove headers? Can you add formatted images to PR comment? Also, a typo: "Refence"